src/components/Verify.js

```diff
@@ -71,7 +71,8 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: theme.palette.action.selected,
   },
   cardContent: {
-    padding: '87% 0 0 0',
+    // padding: '87% 0 0 0',
+    height: '250px',
```

Comment:
We need to keep the aspect ratio of these cards consistent across different screen size, which is why we were using padding. It may be better to apply `position: absolute` to the `Alert` in `OptimizedImage` to get the desired effect.

Reply:
@nmcharlton Hey Nick, I made the requested changes and kept the padding. I also tweaked the message styling based on the current location path. I have the images of the different views below, let me know what you think.

![capture-matching](https://user-images.githubusercontent.com/76817540/164470886-f31a925a-d31b-415f-bfcd-5db928e94c1b.jpeg)

![verify](https://user-images.githubusercontent.com/76817540/164470910-07311a3f-4f02-49dc-b5cb-fae0854b309d.jpeg)

![growers](https://user-images.githubusercontent.com/76817540/164470984-a1d78da2-8630-4f15-8e71-b573b7bb1fc6.jpeg)